vulnerabilities/scraper/alpine_linux.py

```diff
@@ -0,0 +1,42 @@
+import itertools
+import saneyaml
+from urllib.request import urlopen
+from io import BytesIO
+from zipfile import ZipFile
+
+ALPINE_DB_URL = 'https://github.com/alpinelinux/alpine-secdb/archive/master.zip'
+
+
+def alpine_advisories(url):
+    with urlopen(url) as response:
+        with ZipFile(BytesIO(response.read())) as zf:
+            for path in zf.namelist():
+                if path.endswith('main.yaml'):
+                    yield saneyaml.load(zf.open(path))
+
+
+def import_vulnerabilities():
+    vulnerability_package_dicts = []
+    for vulnerability in alpine_advisories(ALPINE_DB_URL):
+        for pkg_details in vulnerability['packages']:
+            package_name = pkg_details['pkg']['name']
+            for version, fixed_cves in pkg_details['pkg']['secfixes'].items():
+                # ['CVE-2016-9932  XSA-200', 'CVE-2016-9815','CVE-????-?????'] after mapping
+                # the split  function to above list
+                all_cves = list(map(lambda x: x.split(), fixed_cves))
+                # it becomes   [['CVE-2016-9932','XSA-200'], ['CVE-2016-9815'],['CVE-????-?????']]
+                all_cves = list(itertools.chain.from_iterable(all_cves))
```

Comment:
If you flatten the list, a non-CVE ID can get attributed to an unrelated vulnerability. For example with this data: https://github.com/alpinelinux/alpine-secdb/blob/master/v3.9/main.yaml#L1935

Reply:
Seeing this data, I see how wrong this approach, I will change this into something else.

Reply:
Flattening the list was bad approach, please check my new approach